test/data/algorithms.js

```diff
@@ -56,5 +56,36 @@ module.exports = [
             //               cFYK/lPtPiVYP4bwg==);{id = 9033}
             "0x91108e1fabbb974406cbdaa90bd975b0b9dc25c38a14b27b1a18943a26eee2d798a79544f519dcae24a164dcfce66c2532034469c1582bf94fb4f89560fe1bc2"
         ]
-    ]
+    ],
+    [
+        "SHA256P256Algorithm",
+        // This test vector generated from the example in RFC5702 using the following Python script:
+        // import dns.rrset
+        //
+        // dnskey = dns.rrset.from_text("example.net.", 3600, "IN", "DNSKEY", "256 3 8 AwEAAcFcGsaxxdgiuuGmCkVImy4h99CqT7jwY3pexPGcnUFtR2Fh36BponcwtkZ4cAgtvd4Qs8PkxUdp6p/DlUmObdk=")
```

Comment:
This happens to be testing for rsa keys (the `8` in `256 3 8` is for rsa signature). You may want to steal examples from https://tools.ietf.org/html/rfc6605#section-6.1 for ecdsa p256 instead

Reply:
Here is script with samples from rfc6605:
``` python
from StringIO import StringIO
import dns.rrset

dnskey = dns.rrset.from_text("example.net.", 3600, "IN", "DNSKEY", "256 3 13 GojIhhXUN/u4v54ZQqGSnyhWJwaubCvTmeexv7bR6edb krSqQpF64cYbcB7wNcP+e+MAnLr+Wi9xMWyQLc8NAA==")

a = dns.rrset.from_text("www.example.net.", 3600, "IN", "A", "192.0.2.1")
buf = StringIO()
a.to_wire(buf)

rrsig = dns.rrset.from_text("www.example.net.", 3600, "IN", "RRSIG", "A 13 3 3600 20100909100439 20100812100439 55648 example.net.  qx6wLYqmh+l9oCKTN6qIc+bw6ya+KJ8oMz0YP107epXA yGmt+3SNruPFKG7tZoLBLlUzGGus7ZwmwWep666VCw==")
signature = rrsig[0].signature
rrsig[0].signature = ''
signdata = rrsig[0].to_digestable() + buf.getvalue()

print ('0x' + dnskey[0].to_digestable().encode('hex'), '0x' + signdata.encode('hex'), signature.encode('hex'))
```

Reply:
Good catch, thank you!

Reply:
yeah the comment was never updated, I was lazy